packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -69,26 +75,17 @@ export function menuItemTemplate<T extends FASTMenuItem>(
                 </div>
             `
         )}
-        ${when(
-            x => x.expanded,
-            html<T>`
-                <${anchoredRegionTag}
-                    :anchorElement="${x => x}"
-                    vertical-positioning-mode="dynamic"
-                    vertical-default-position="bottom"
-                    vertical-inset="true"
-                    horizontal-positioning-mode="dynamic"
-                    horizontal-default-position="end"
-                    class="submenu-region"
-                    dir="${x => x.currentDirection}"
-                    @loaded="${x => x.submenuLoaded()}"
-                    ${ref("submenuRegion")}
-                    part="submenu-region"
-                >
-                    <slot name="submenu"></slot>
-                </${anchoredRegionTag}>
-            `
-        )}
+        <span
+            ?hidden="${x => !x.expanded}"
+            class="submenu-container"
+            part="submenu-container"
+            ${ref("submenuContainer")}
+        >
```

Comment:
Can we dig in here on the *value* of an internal element like this?

Reply:
Since the only thing we can confirm about the submenu is that it's an element with a `role` of `menu`, I think it's safer and less obtrusive to handle the positioning in the menu-item's shadow rather than on the slotted submenu element itself.

Here are some alternatives:
* Setting the positioning styles with the `style` attribute on the slotted submenu can cause conflicts with specificity and author intent.
* Attaching a stylesheet to the nested element would address the specificity problem, but it would only be possible if the submenu is guaranteed to have a FAST controller (`$fastController`) to attach the styles.

Reply:
I agree this is probably a preferred API, though I'm certainly curious of others thoughts. Ideally we can document the intent as part of the readme, or somewhere else to help convey this. I think given the alternatives, this is the best method of providing positioning without needing to get prescriptive with whats slotted in.